### ISSUE TITLE
Could not find a declaration file for module.

### ISSUE THREAD

**Issue by mzavattaro** [uidotdev/usehooks#200]
I'm trying to utilise this library within a NextJS App directly project. However, I receive the following error:

```
Could not find a declaration file for module '@uidotdev/usehooks'. '/Users/michaelzavattaro/Github/kachingme/node_modules/.pnpm/@uidotdev+usehooks@2.0.1_react-dom@18.2.0_react@18.2.0/node_modules/@uidotdev/usehooks/index.js' implicitly has an 'any' type.

Try `npm i --save-dev @types/uidotdev__usehooks` if it exists or add a new declaration (.d.ts) file containing `declare module '@uidotdev/usehooks';`ts(7016)
```

My current setup is:
- "@uidotdev/usehooks": "^2.0.1",
- "next": "13.4.6",
- "react": "18.2.0",

I installed via `pnpm add @uidotdev/usehooks`

**Comment by xg4** [comment]
It is related to issue #185

**Comment by tylermcginnis** [comment]
Should be fixed with #206
